numba_cuda/numba/cuda/__init__.py

```diff
@@ -7,12 +7,15 @@
 import warnings
 import sys
 
+from numba_cuda._version import __version__
+
 # Re-export types itself
 import numba.cuda.types as types
 
 # Re-export all type names
 from numba.cuda.types import *
 
+HAS_NUMBA = importlib.util.find_spec("numba") is not None
```

Comment:
I think we might need to change the implementation of this in future, so having it centralized here is convenient.